packaging/README.md

```diff
@@ -14,6 +14,18 @@ You must also have an environment variable set with the id of the key to use. Vi
 Login: openhie
 PPA: release
 
+# Bundled Release
+
+A bundled release will ensure all the relevant dependencies are downloaded and bundled into a built version of the OpenHIM core. Only the relevant scripts needed to run the OpenHIM core is added to the bundled release.
+
+To create a new build release execute to the below command. This does assume that your Linux distribution has the `zip` module installed
```

Comment:
it should be - "To create a new build release execute the below command"

Reply:
done in ec9ae44